### DISCUSSION TITLE
Data assets not fitting into memory

### DISCUSSION THREAD

**Discussion by akharlamov** [dagster-io/dagster#18502] [Q&A]
Hey! 

I build an ETL process that operates on datasets in the 10-100GiB regions. It is not feasible to load them to RAM, but PySpark is not the way to handle them either. In Airflow I would just use custom operators, 

I would like a data flow orchestration system to be aware of data assets, hence I look to Dagster. 

Is there any ETL framework integration/IO manager capable of helping with that?

Thank you in advance

**Comment by yuhan** [comment]
How about https://docs.dagster.io/integrations/embedded-elt?

Similar to Airflow custom operator, [Dagster Pipes](https://docs.dagster.io/guides/dagster-pipes) is built for invoking external execution. Here's an example for databricks integration: https://docs.dagster.io/guides/dagster-pipes/databricks
